# SyncedMonoStream.msg
uint64 timestamp
int32 enc1
int32 enc2
int32 enc3
float64 x
float64 y

